test/verify/check-services

```diff
@@ -188,6 +189,281 @@ WantedBy=default.target
         b.go('#/foo')
         b.wait_text('#exists', 'false')
 
+    def testCreateTimer(self):
+        m = self.machine
+        b = self.browser
+
+        def svc_sel(service):
+            return 'tr[data-goto-unit="%s"]' % service
+        def wait_systemctl_timer(time):
+            with testvm.Timeout(seconds=20, error_message="Timeout while waiting for systemctl list-timers"):
+                m.execute("sleep 1")
```

Comment:
I thought if we called `systemctl list-timer` 's in next line after 1 sec, then it can have the updated values. Let me test it locally in centos and fedora-atomic without the 1 sec and see if it works.